textproc/txt2html/Portfile

```diff
@@ -1,12 +1,12 @@
 PortSystem 1.0
```

Comment:
please add another commit with non-functional changes where you fix the whitespace issues (i.e., with spaces in multiples of four).